internal/start_command.go

```diff
@@ -31,3 +31,35 @@ func StartCommand(command []string) (waitFunc func(), stdout io.ReadCloser) {
 		}
 	}, stdoutResult
 }
+
+func ApplyCommand(path string, args []string, stdin io.ReadCloser) error {
+	c := exec.Command(path, args...)
+	if stdin != nil {
+		c.Stdin = stdin
+	}
+	stderrResult, err := c.StderrPipe()
+	if err != nil {
+		tracelog.ErrorLogger.PrintError(err)
+		var errorString string
+		if errorBytes, err := ioutil.ReadAll(stderrResult); err == nil {
+			errorString = string(errorBytes)
+		}
+		tracelog.ErrorLogger.Println(errorString)
+		return err
+	}
+	err = c.Start()
+	tracelog.ErrorLogger.FatalOnError(err)
+	if err != nil {
+		return err
+	}
+	tracelog.InfoLogger.Println("waiting on command")
+	if err := c.Wait(); err != nil {
```

Comment:
We should read from stderrResult, otherwise Command may be blocked forever
Consider using https://golang.org/pkg/os/exec/#Cmd.CombinedOutput